### PR TITLE
Remove info graphs

### PR DESCRIPTION
**Pull request by alegnaaived**
I moved all the charts to one folder outside modules folders, and it's necessary for new fixes to check eslint rules in others charts (BarGraph and DotsGraph).

### CODE REVIEW

File: src/charts/BarStackHorizontal.jsx [hunk 1]
```diff
@@ -30,38 +31,42 @@ export default withTooltip(
...
+      });
       return transformedData;
-    }
+    };
 
     function comparingValues(a, b) {
       return a - b;
     }
 
+    let data = [prepareData(dataJSON, labelY)].slice(0);
```
Comment (erikasv) [inline src/charts/BarStackHorizontal.jsx]: This is unnecessary, data should be just `prepareData(dataJSON, labelY)`, and so, the next line would be just `Object.keys(data)`

File: src/charts/BarStackHorizontal.jsx [hunk 2]
```diff
@@ -111,44 +119,45 @@ export default withTooltip(
               xScale={xScale}
               yScale={yScale}
               zScale={zScale}
-              onClick={data => event => {
-                if (!events) return;
-                alert(`clicked: ${JSON.stringify(data)}`);
-                // actualizarSubArea(data.key);
-              }}
-              onMouseLeave={data => event => {
+              // onClick={data => (event) => {
```
Comment (erikasv) [inline src/charts/BarStackHorizontal.jsx]: you should remove unused code

File: src/charts/BarStackHorizontal.jsx [hunk 3]
```diff
@@ -14,7 +15,7 @@ export default withTooltip(
     width,
     height,
     graphTitle,
-    events = false,
+    // events = false,
```
[the remark this answers is not in the document]
Comment (alegnaaived) [reply src/charts/BarStackHorizontal.jsx]: Solved

File: src/charts/BarStackHorizontal.jsx (hunk 2 above)
Comment (alegnaaived) [reply src/charts/BarStackHorizontal.jsx]: Solved

File: src/charts/BarStackHorizontal.jsx (hunk 1 above)
Comment (alegnaaived) [reply src/charts/BarStackHorizontal.jsx]: After pair review, it is now solved

**Pull request merged**